Dockerfile

```diff
@@ -9,20 +9,24 @@ WORKDIR /app
 
 COPY poetry.lock /app/
 COPY pyproject.toml /app/
-COPY ${package}/ /app/${package}
 
-RUN python3.9 -m pip install poetry \
- && python3.9 -m poetry config virtualenvs.create false \
- && python3.9 -m poetry install --without dev \
- && python3.9 -m poetry export -f requirements.txt --output requirements.txt --without-hashes
+RUN python3.9 -m pip install poetry==1.4.2
+RUN python3.9 -m poetry config virtualenvs.create false
+RUN python3.9 -m poetry install --without dev --no-root
+RUN python3.9 -m poetry export -f requirements.txt --output requirements.txt --without-hashes
 
 # run tests
-COPY tests /app/tests
+COPY ${package}/ /app/${package}
+COPY tests /app/${package}/tests
+
+ENV PYTHONPATH /app/${package}
+WORKDIR /app/${package}
 
 RUN mkdir /htmlcov
-RUN pip3 install coverage
-RUN python3 -m coverage run tests/test_arcaflow_plugin_template_python.py
-RUN python3 -m coverage html -d /htmlcov --omit=/usr/local/*
+RUN python3.9 -m pip install coverage==7.2.7
+RUN python3.9 -m coverage --version
```

Comment:
why calling `coverage --version`?

Reply:
That was an artifact of testing; removed.